recipes/general_robotics_toolbox/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: general_robotics_toolbox
+  version: 0.7.1
+
+source:
+  url: https://github.com/rpiRobotics/rpi_general_robotics_toolbox_py/archive/refs/tags/0.7.1.tar.gz
```

Comment:
If you use `jinja` here for the version number, then the conda-forge bots can pick up new releases automatically. 

E.g. at the beginning of the recipe, use `{% set version = "0.7.1" %}`, and then `.../tags/{{ version }}.tar.gz` here.

Reply:
I have made the requested changes